docs/walkthrough/addNewPage.md

```diff
@@ -1,47 +1,62 @@
 Walkthrough: Adding a New Page
 ===========================
 
-The following is for creating a new page called "**example**."
+The following is for creating a new page called "**basicPage**."
 
 1. Create a folder named `example` inside the `components/pages` folder.
 1. Create 4 files in the new folder. See the individual example files for more details and comments inline.
-    - [example.container.js](/src/components/pages/_example/example.container.js) - maps redux and epic actions and selectors to the props for the page
-    - [example.js](/src/components/pages/_example/example.js) - main component for the page
-    - [example.scss](/src/components/pages/_example/example.scss) - styles for the page
-    - [example.test.js](/src/components/pages/_example/example.test.js) - basic rendering test
-1. Add the new page's container to the [components/pages/index.js](/src/components/pages/index.js) file.
+    - [basicPage.container.js](/src/walkthrough/components/pages/basicPage/basicPage.container.js) - maps redux and epic actions and selectors to the props for the page
+    - [basicPage.js](/src/walkthrough/components/pages/basicPage/basicPage.js) - main component for the page
+    - [basicPage.scss](/src/walkthrough/components/pages/basicPage/basicPage.scss) - styles for the page
+    - [basicPage.test.js](/src/walkthrough/components/pages/basicPage/basicPage.test.js) - basic rendering test
+1. Add the new page's container to the [components/pages/index.js](/src/walkthrough/components/pages/index.js) file.
     ```js
-    export * from './example/example.container';
+    export * from './basicPage/basicPage.container';
     ```
 1. (Optional)  Add an SVG icon for the new page. See [utilities/README.md](../utilities/README.md) for more information.
     - Note that existing SVGs can be used as well.
 1. Add the page name to the translations file, [translations.json](../../public/locales/en/translations.json). [i18next][i18next] is used for internationalization.
     ```json
     "tabs": {
-      "template": "Template",
+      "template": "Example",
```

Comment:
This isn't your fault... but would you change this to 
```
"example": "Example",
```

Reply:
done